### PR TITLE
dogehouse: fix and add stuff

### LINKED ISSUE #2369: Chat automentions users when you type their username
https://user-images.githubusercontent.com/57961822/116165421-22c16080-a6fc-11eb-80ad-79640421f0b0.mov

### PR DESCRIPTION
**Pull request by amitojsingh366**
- revert 8cf3df77aa135f92181f661da6e7bb930124681d *(fixes #2369)*
- add commit signing to commitzen *(you can now use `yarn commit:signed` to sign your commits)*
- add auto labeler to automatically label prs *(see it in action [here](https://github.com/amitojsingh366/dogehouse/pull/5))*
- return username along with tokens from `/bot/auth`
- fixed mute icon staying active color even though user is deafened
- fix linting issues
- fix chat icon size on smaller screens

### CODE REVIEW

**Review by BronzW — APPROVED** [review]

**Comment by nadirabbas** [comment]
is this ready to be merged?

**Comment by amitojsingh366** [comment]
> 
> 
> is this ready to be merged?

yes ready now

**Comment by nadirabbas** [comment]
Don't revert my commit yet, it was done for a reason and to fix some behaviors. I will fix the bug it introduced a bit differently.

**Comment by amitojsingh366** [comment]
> 
> 
> Don't revert my commit yet, it was done for a reason and to fix some behaviors. I will fix the bug it introduced a bit differently.

you shouldn't be pushing commits that break other things *(remembers when he broke dogehouse with deafen)* , lets keep the revert until you push up the full fix

**Comment by nadirabbas** [comment]
LOL, I didn't push it deliberately, I didn't know it caused a bug. you can revert your fix as I will be pushing the fix tonight.

**Comment by benawad** [comment]
keep the revert in, the chat is annoying pinging everyone atm, nadir can add it back when he does his fix

**Comment by nadirabbas** [comment]
I am just pushing the fix, but yearh @amitojsingh366 you can keep it in then

**Comment by nadirabbas** [comment]
the fix: 1766364

**Comment by amitojsingh366** [comment]
> 
> 
> the fix: [1766364](https://github.com/benawad/dogehouse/commit/1766364c6941f4333f6a1709ce7e2866d01a1773)

nadir bad

File: kibbeh/src/shared-hooks/useSetMute.ts [hunk 1]
```diff
@@ -3,21 +3,23 @@ import { useDeafStore } from "../global-stores/useDeafStore";
...
-};
-
 export const setMute = (conn: Wrapper, value: boolean) => {
   const { setInternalDeaf, deafened } = useDeafStore.getState();
   // auto undeafen on unmute
   if (deafened) {
     setInternalDeaf(false);
     conn.mutation.setDeaf(false);
+    useMuteStore.getState().setInternalMute(false, false);
+    conn.mutation.setMute(false);
```
Comment (benawad) [inline kibbeh/src/shared-hooks/useSetMute.ts]: I don't want this

**Review by benawad — CHANGES_REQUESTED** [review]

File: kibbeh/src/shared-hooks/useSetMute.ts (hunk 1 above)
Comment (amitojsingh366) [reply kibbeh/src/shared-hooks/useSetMute.ts]: removed it

**Comment by overlisted** [comment]
you converted some json files to crlf

**Pull request merged**